created, tested, and merged new feature

rtstewart opened the pull request.

rtstewart committed:
changed secondfile.txt in new local branch "testfeature"

 On branch testfeature
 Changes to be committed:
	modified:   secondfile.txt

rtstewart committed:
creating merge conflict to practice resolution

 On branch master
 Your branch is up-to-date with 'origin/master'.

 Changes to be committed:
	modified:   secondfile.txt

rtstewart committed:
Merge branch 'testfeature'

 Conflicts:
	secondfile.txt

 It looks like you may be committing a merge.
 If this is not correct, please remove the file
	.git/MERGE_HEAD
 and try again.

 Please enter the commit message for your changes. Lines starting
 with '' will be ignored, and an empty message aborts the commit.
 On branch master
 Your branch is ahead of 'origin/master' by 1 commit.
   (use "git push" to publish your local commits)

 All conflicts fixed but you are still merging.

 Changes to be committed:
	modified:   secondfile.txt

slambonium commented:
Hey, thanks for doing this. Can you add "Appending this to second file.txt locally and pushing to add to the pull request?"

rtstewart committed:
adding a commit to see it get added to pull request just made

 On branch master
 Your branch is up-to-date with 'origin/master'.

 Changes to be committed:
	modified:   secondfile.txt

rtstewart commented:
Done.
